app/components/dashboard/DcPanelAnalyze.vue

```
@@ -0,0 +1,397 @@
+</script>
+
+<template>
+  <div id="panel_ana" class="bg-theme-col2 dc_panel drag" v-show="options.is_show" style="display: none;">
+    <div class="chart-title-wrap">
+      <span class="chart-title text-theme-col2" v-html="options.title"></span>
+      <span class="ui-icon ui-icon-circle-close sp_icon btn_close" @click="handleClose"></span>
+    </div>
+    <div class="clearfix"></div>
+
+    <div id="ana_diff_ls">
+      <table id="tbl_ana" border="1" bordercolor="#b0b0b0" style="float:left;">
```

ESLint

[error] 'style' should be on a new line.

(vue/max-attributes-per-line)